### PR TITLE
midi_arp: reset midi after all previous notes are released

### PR DESCRIPTION
**Pull request by samlletas**
Currently MIDI mode resets the playhead as soon as any new note is pressed, I thought it would be useful to play a chord and be able to for example change one note to create some variation without fully resetting the pattern, to allow this, this PR adds an additional condition to make the playhead reset only after all of the previous notes have been released (similar to BlueARP when configured to reset on 'key').

### CODE REVIEW

**Review by JoepVanlier — CHANGES_REQUESTED** [review]
Thank you! That makes a lot more sense yeah.

Please bump the version number at the top of the script, update the changelog entry and I'd be happy to merge this.

For the changelog entry, keep it to one line ideally (you can put your user name or handle in brackets if you want).

**Comment by samlletas** [comment]
Thanks for taking a look at this!

I have pushed a new commit with the version bump details, please let me know if further changes are needed.

File: saike_midi_arp/saike_midi_arp.jsfx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 desc:Saike MIDI ARP (beta)
 tags: midi arpeggiator
-version: 0.34
+version: 0.35
 author: Joep Vanlier
-changelog: Bump version number.
+changelog: When set to MIDI mode, reset the pattern only after all previous notes are relased.
```
Comment (JoepVanlier) [inline saike_midi_arp/saike_midi_arp.jsfx:5]: Minor nitpick, but there's a typo. Should be `released`.

**Review by JoepVanlier — APPROVED** [review]
Looks good! There's a small type in the changelog entry, but it's OK.

File: saike_midi_arp/saike_midi_arp.jsfx (hunk 1 above)
Comment (samlletas) [reply saike_midi_arp/saike_midi_arp.jsfx:5]: My apologies😅, I see the PR is already merged/closed but let me know if you want me to fix it.
Thanks for merging these changes!

**Pull request merged**